src/client/app/reducers/readings.js

```diff
@@ -16,7 +16,12 @@ import * as readingsActions from '../actions/readings';
  * @type {State~Readings}
  */
 const defaultState = {
```

Comment:
On this branch, when the page loads for the first time, readings for both the line chart and bar chart (default options, 30 day bar chart width, etc.) are called. This lets the user switch toggle between bar and line with no delay. The other actions are called when users request specific non-default readings.